packages/java-edition/src/mcfunction/checker/index.ts

```diff
@@ -242,6 +248,33 @@ function nbtChecker(
 	}
 }
 
+function getListLikeChild(typeDef: mcdoc.McdocType): mcdoc.McdocType | undefined {
```

Comment:
Should this be changed to take a `SimplifiedMcdocType`?

Reply:
The return value no, because the item of a list is not simplified. The parameter could be though.